control-plane/csi-driver/controller/src/controller.rs

```diff
@@ -266,19 +266,20 @@ impl rpc::csi::controller_server::Controller for CsiControllerSvc {
         let mut allowed_nodes: HashSet<String> = HashSet::new();
         let mut preferred_nodes: HashSet<String> = HashSet::new();
         let mut inclusive_label_topology: HashMap<String, String> = HashMap::new();
+        // List of supported topology keys are:-
+        // {
+        //     openebs.io/nodename
+        //     kubernetes.io/hostname
+        // }
```

Comment:
Do we need this, isn't it "self-documented" below?

Reply:
Yeah we don't